.github/workflow/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should we add the regex for version branches here since this is using `semantic-release`?